qiskit_experiments/library/calibration/fine_amplitude.py

```diff
@@ -32,74 +32,76 @@
-      if the square-root of X gate is added before the repeated gates. This is decided for
-      the user in :meth:`set_schedule` depending on whether the sx gate is included in the
-      experiment.
-
-    Users can call :meth:`set_schedule` to conveniently set the schedule and the corresponding
-    experiment and analysis options.
+    # section: overview
+
+        The :class:`FineAmplitude` calibration experiment repeats N times a gate with a pulse
+        to amplify the under-/over-rotations in the gate to determine the optimal amplitude.
+        The circuits that are run have a custom gate with the pulse schedule attached to it
+        through the calibrations. The circuits are therefore of the form:
+
+        .. parsed-literal::
+
+                       ┌─────┐       ┌─────┐ ░ ┌─┐
+                  q_0: ┤ Cal ├─ ... ─┤ Cal ├─░─┤M├
+                       └─────┘       └─────┘ ░ └╥┘
+            measure: 1/════════ ... ════════════╩═
+                                                0
+
+        Here, Cal is the name of the gate which will be taken from the name of the schedule.
+        The user can optionally add a square-root of X pulse before the Cal gates are repeated.
+        This square-root of X pulse allows the analysis to differentiate between over rotations
+        and under rotations in the case of pi-pulses. Importantly, the resulting data is analyzed
+        by a fit to a cosine function in which we try and determine the over/under rotation given
+        an intended rotation angle per gate which must also be specified by the user. The steps
+        to run a fine amplitude calibration experiment are therefore
+
+        .. code-block:: python
+
+            qubit = 3
+            amp_cal = FineAmplitude(qubit)
+            amp_cal.set_schedule(
+                schedule=x90p,
+                angle_per_gate=np.pi/2,
+                add_xp_circuit=False,
+                add_sx=False
+            )
+            amp_cal.run(backend)
+
+        Note that the schedule and angle_per_gate could have been set by independently calling
+        :meth:`set_experiment_options` for the schedule and :meth:`set_analysis_options` for
+        the angle_per_gate.
+
+        Error amplifying experiments are most sensitive to angle errors when we measure points along
+        the equator of the Block sphere. This is why users should insert a square-root of X pulse
+        before running calibrations for :math:`\pm\pi` rotations. Furthermore, when running
+        calibrations for :math:`\pm\pi/2` rotations users are advised to use an odd number of
+        repetitions, e.g. [1, 2, 3, 5, 7, ...] to ensure that the ideal points are on the equator
+        of the Bloch sphere. Note the presence of two repetitions which allows us to prepare the
+        excited state. Therefore, add_xp_circuit = True is not needed in this case.
+
+        **Summary of experiment options**

```

Comment:
Please move them to `_default_experiment_options` with type hints.

Reply:
See here: eac66ce